### PR TITLE
[fastlane] Switched to use built-in command runner

### PR DESCRIPTION
**Pull request by KrauseFx**
Updated Fastfile to use built-in commander runner

### CODE REVIEW

File: fastlane/fastlane/Fastfile [hunk 1]
```diff
@@ -185,8 +185,8 @@ private_lane :print_release_changelog do |options|
   tool_name = options[:tool]
   old_version = options[:old_version]
 
-  cmd = "git log --pretty='* %s' #{tool_name}/#{old_version}...HEAD --no-merges .."
-  puts "Changes since release #{old_version}:\n" + `#{cmd}`.gsub("\n\n", "\n")
+  changes = sh("git log --pretty='* %s' #{tool_name}/#{old_version}...HEAD --no-merges ..").gsub("\n\n", "\n")
```
Comment (mfurtak) [inline fastlane/fastlane/Fastfile]: I believe this change behaves differently than the original, and will not work. 😞 

```ruby
task :blah do
  value = sh('pwd')
  puts "Value: #{value}"
end
```

```
$ rake blah
pwd
/Users/mfurtak/Documents/Programming/Ruby/fastlane
Value: true
```

`sh()` always prints the the command and the output, but returns `true`/`false` 😫

Comment (KrauseFx) [inline fastlane/fastlane/Fastfile]: Oh, so we could use `Helper.backticks`, right?

**Comment by mfurtak** [comment]
LGTM

File: fastlane/fastlane/Fastfile (hunk 1 above)
Comment (mfurtak) [inline fastlane/fastlane/Fastfile]: My mistake - see my edit above. If you've tested it, go for it! :smile:

**Comment by KrauseFx** [comment]
Ah, so this is good to merge, since it's the rake's `sh` action, right? 👍

**Comment by KrauseFx** [comment]
Just tested this, and it works as expected. Nice thing is, that it now prints out the executed command, making it easier to debug things 👍

**Pull request merged**